packages/rocketchat-api/server/helpers/deprecationWarning.js

```diff
@@ -0,0 +1,10 @@
+RocketChat.API.helperMethods.set('deprecationWarning', function _deprecationWarning({ endpoint, versionWillBeRemove, response }) {
+	const warningMessage = `The endpoint ${ endpoint } is deprecated and will be removed after version ${ versionWillBeRemove }`;
```

Comment:
I apologize for being so picky, but can the endpoint be surrounded in quotes? `The endpoint "${ endpoint }" is..`

Reply:
Of course @graywolf336! I'll do it. =)